examples/energy-management-app/esp32/sdkconfig.optimize.defaults

```diff
@@ -1,5 +1,5 @@
 #
-#    Copyright (c) 2023 Project CHIP Authors
+#    Copyright (c) 2024 Project CHIP Authors
```

Comment:
Why?

Reply:
Same comment as above

Reply:
I have changed these 2023-2024 as you suggested below